### PR TITLE
Added lighthouse command

### LINKED ISSUE #145: [Feature]: Add lighthouse command
### To which use case does your feature request relate?

_No response_

### Is your feature request related to a problem? Please describe

Currently, we have to execute the lighthouse command manually.

### Describe the solution you'd like

Add a command that improves this experience.

### Describe alternatives you've considered

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Slartibartfass2**
Closes #145

## What I have made

- I added a lighthouse command, see the description in the README

## Checklist

Either tick or cross out the items that do not apply (using \~\~example text\~\~) and give a reason why the item does not apply.

- [x] I have updated the documentation accordingly and commented my code
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~ (no new logic)
- [ ] (for reviewer) I have checked the implementation against the requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +68,13 @@ End-to-end tests are used to test the functionality of the app as a whole. They
 ```bash
 npm run test:e2e
 ```
+
+## Lighthouse
+
+To run a Lighthouse audit on the app, you can use:
+
+```bash
+npm run lighthouse -- http://localhost:4173/path
```
Comment (Merseleo) [inline README.md:83]: If I run this command, I get "lighthouse command not found", so probably add a hint, that lighthouse first need to be installed correctly (with a link to the installation or so).

**Review by Merseleo — CHANGES_REQUESTED** [review]
I like the badge for the use cases, but a question of interest: why did you need to run a lighthouse audit locally?

File: README.md (hunk 1 above)
Comment (Slartibartfass2) [reply README.md:83]: I added a description in the README

**Comment by Slartibartfass2** [comment]
> I like the badge for the use cases, but a question of interest: why did you need to run a lighthouse audit locally?

I usually like to run it locally to get faster feedback after I implemented a new feature. Also I can just run it for one page and don't have to wait for the CI to audit all pages.

File: README.md (hunk 1 above)
Comment (Merseleo) [reply README.md:83]: Thank you, but when I run the frontend, it starts on port 5173 and not 4173, so is the port you provided in your example intended or not?

Comment (Slartibartfass2) [reply README.md:83]: 4173 is the preview. I usually test it when completely built and served.
But this is just an example.

Comment (Merseleo) [reply README.md:83]: Makes sense.

**Review by Merseleo — APPROVED** [review]

**Pull request merged**